.github/workflows/pg-migtests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   run-pg-migration-tests:
     strategy:
       matrix:
-        version: [2.21.0.0-b545, 2.16.9.0-b67, 2.18.7.0-b30, 2.20.2.2-b1]
+        version: [2.21.0.0-b545, 2.18.7.0-b30, 2.20.2.2-b1]
```

Comment:
Shall we do this change in a separate PR for both the workflows MySQL and PG?
Or 
Do it for both the workflows and mention this in the commit-msg to the main branch and also the reason for removing this version (end of maintenance).

Reply:
Creating a separate PR.